installer/api.py

```diff
@@ -56,14 +57,12 @@ def sendApiPOST(retCode: int=0):
 		headers = {"Content-type": "application/json"}
 
 		# Establish a connection
-		url = API_URL.split("/", maxsplit=1)
-		path = f"/{url[1]}"
-		url = url[0]
-		conn = http.client.HTTPSConnection(url, timeout=2, context=ssl._create_unverified_context()) # Unverified context because url does not have an ssl certificate
+		parsedUrl = urlparse(API_URL)
+		conn = http.client.HTTPSConnection(parsedUrl.hostname, parsedUrl.port, timeout=2)
 
 		try:
 			# Send the POST request
-			conn.request("POST", path, params, headers)
+			conn.request("POST", path, body=params, headers=headers)
```

Comment:
Not needed, since `request` method takes two positional params (`method` and `url`), and the rest are keyword:
![image](https://github.com/user-attachments/assets/94e44d54-e986-407b-b40c-5039b8c83bdc)
So, the first two params **should** be positional, and all the rest keyword.